sugoi-api-core/src/main/java/fr/insee/sugoi/core/seealso/SeeAlsoService.java

```diff
@@ -0,0 +1,64 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+package fr.insee.sugoi.core.seealso;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SeeAlsoService {
+
+  protected static final Logger logger = LogManager.getLogger(SeeAlsoService.class);
+
+  @Autowired private MapProtocolSeeAlsoDecorator mapProtocol;
+
+  /**
+   * Take a seeAlso string described as "url|subobject|attributetoupdate" and return the suboject
+   * value of the resource fetch at url.
+   *
+   * <p>url is the location of the resource to parse (ex : http://example.org/ex). The url contain
+   * protocol and may contain a port.
+   *
+   * <p>subobject is the description of how to get the String or List<String> from the fetched
+   * resource
+   *
+   * @param seeAlso
+   * @return the value retrieved from ldap or http. It can be a String or a List<String>. If an
+   *     error occured while fetching the resource or parsing it, null is returned.
+   */
+  public Object getSeeAlsoFieldValue(String seeAlso) {
+    try {
+      String url = seeAlso.split("\\|")[0];
+      String subobject = seeAlso.split("\\|")[1];
+      String protocol = url.split(":")[0];
+      return mapProtocol.getResourceFromUrl(protocol, url, subobject);
+    } catch (Exception e) {
+      logger.error(e);
```

Comment:
Coyuld you log a message with this exception (to identify this is caused by the seealsoservice)